<launch>
    <rosparam file="$(find rm_hw)/test/test_imu.yaml" command="load"/>

    <!-- load the controllers -->
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
          output="screen" args="
        controllers/imu_sensor_controller
"/>

    <node name="rm_hw" pkg="rm_hw" type="rm_hw" respawn="false"/>

</launch>
